app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -180,6 +180,24 @@
   }
 }
 
+.l-header__content {
+  position: fixed;
+  background: $color-green-primary;
+  z-index: 150;
```

Comment:
Is there any way we can make keep this and the other z-index on line 3 under 100? Based on what we saw yesterday with the Dragon browsing tech?